ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -614,3 +614,71 @@ def test_tensorflow_Minimum(
         x=np.asarray(x[0], dtype=input_dtype[0]),
         y=np.asarray(x[1], dtype=input_dtype[1]),
     )
+
+
+# Less
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
+        ),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+    as_variable=helpers.array_bools(num_arrays=2),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.Less"
+    ),
+    native_array=helpers.array_bools(num_arrays=2)
```

Comment:
Since we're using `@handle_cmd_line_args`, we can omit `as_variable`, `native_array` as this is handled by the us.

Reply:
Solved in https://github.com/unifyai/ivy/pull/3714/commits/33e92c4631aa6b6dfb4a94e162e303e1d0caa5fc